resources/src/operation/mod.rs

```diff
@@ -0,0 +1,220 @@
+use amount::Amount;
+use asset::AssetIdentifier;
+use serde::{de, Deserialize, Deserializer};
+use offer::PriceRatio;
+mod create_account;
+use self::create_account::CreateAccountFields;
+
+/// Operations are objects that represent a desired change to the ledger: payments, offers to
+/// exchange currency, changes made to account options, etc. Operations are submitted to the
+/// Stellar network grouped in a Transaction.
+
+#[derive(Debug, Serialize)]
+pub struct Operation {
+    id: i64,
+    paging_token: String,
+    type_i: u32,
+    operation_detail: OperationDetail,
```

Comment:
aren't they called types in the stellar documentation? I'd rather use that nomenclature here.

If we do use `detail` let's just call it `detail` instead of prepending `operation_` to it. Makes it a little shorter.